server/build.gradle

```diff
@@ -27,6 +27,8 @@ dependencies {
     implementation("io.vertx:vertx-auth-jwt")
     implementation("io.vertx:vertx-opentelemetry")
     implementation("io.vertx:vertx-micrometer-metrics")
+    implementation("io.vertx:vertx-junit5:$vertx_version")
```

Comment:
for testing purpose ? use testImplementation instead of implementation.

Reply:
removed, was redundant just checked